index.html

```diff
@@ -33,21 +33,31 @@
     };
   });
 
+const trusted = /^127\.0\.0\.1(?::|$)|localhost(?::|$)|(?:app\.|local\.)?webaverse\.com|webaverse\.link/
+const github = /githubpreview\.dev$/
```

Comment:
Can we be more careful with this regex? This would also match `hack-githubpreview.dev`.

Reply:
We can probably do the full ```^(?:.+\.)*```.